### PR TITLE
Initial minimal doc for images

### PR DESCRIPTION
**Pull request by jfdenise**
Minimal doc that will evolve when adding more content.

### CODE REVIEW

**Comment by jfdenise** [comment]
@jmesnil initial images doc.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,73 @@
...
+EAP 8 runtime images:
+
+* `jboss-eap-8-tech-preview/eap8-openjdk11-runtime-openshift-rhel8`
+
+* `jboss-eap-8-tech-preview/eap8-openjdk17-runtime-openshift-rhel8`
+
+
+# Relationship with EAP 7.x images
+
+As opposed to the EAP 7.x S2I (Source-To-Image) builder image that is containing an EAP server, the new builder image
```
Comment (jmesnil) [inline README.md]: s/is containing/contains

File: README.md [hunk 2]
```diff
@@ -1,2 +1,73 @@
...
+As opposed to the EAP 7.x S2I (Source-To-Image) builder image that is containing an EAP server, the new builder image 
+is a generic builder allowing to build image for any EAP 8 releases.
+
+# EAP 8 - S2I builder image
+
+The S2I builder image contains all you need to execute a Maven build of your project during an S2I build phase on OpenShift.
+
+## S2I build workflow
+
+The builder image expects that, during the Maven build, an EAP 8 server containing the deployment is being provisioned (by default in `<application project>/target/server` directory). This provisioned server
```
Comment (jmesnil) [inline README.md]: s/expects/requires

 (seems stronger)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,73 @@
...
+# EAP 8 - S2I builder image
+
+The S2I builder image contains all you need to execute a Maven build of your project during an S2I build phase on OpenShift.
+
+## S2I build workflow
+
+The builder image expects that, during the Maven build, an EAP 8 server containing the deployment is being provisioned (by default in `<application project>/target/server` directory). This provisioned server 
+is installed by the image in `/opt/server`. Making the generated application image runnable.
+
+In order to provision a server during the build phase you must integrate (generally in a profile named `openshift` profile) an execution of the  [EAP Maven plugin](https://github.com/jbossas/eap-maven-plugin/).
```
Comment (jmesnil) [inline README.md]: should we mention the `package` goal?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,73 @@
...
+
+For more information on the EAP cloud feature-pack features, check [this documentation](https://github.com/jbossas/eap-cloud-galleon-pack/blob/main/README.md).
+
+## Backward compatible S2I workflow
+
+In case you want to keep your existing project that used to work with the legacy EAP 7.x S2i builder image, you can use a set of environment variables 
+to initiate a server provisioning prior to execute the Maven build of your application:
+
+* `GALLEON_PROVISION_FEATURE_PACKS`: Comma separated lists of Galleon feature-packs, for example: 
+`GALLEON_PROVISION_FEATURE_PACKS=org.jboss.eap:wildfly-ee-galleon-pack,org.jboss.eap.cloud:eap-cloud-galleon-pack:8.0.0.Final-redhat-00001`
```
Comment (jmesnil) [inline README.md]: do you need to specify the `version` of `org.jboss.eap:wildfly-ee-galleon-pack`?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,73 @@
...
+
+## Backward compatible S2I workflow
+
+In case you want to keep your existing project that used to work with the legacy EAP 7.x S2i builder image, you can use a set of environment variables 
+to initiate a server provisioning prior to execute the Maven build of your application:
+
+* `GALLEON_PROVISION_FEATURE_PACKS`: Comma separated lists of Galleon feature-packs, for example: 
+`GALLEON_PROVISION_FEATURE_PACKS=org.jboss.eap:wildfly-ee-galleon-pack,org.jboss.eap.cloud:eap-cloud-galleon-pack:8.0.0.Final-redhat-00001` 
+
+* `GALLEON_PROVISION_LAYERS`: Comma separated lists of Galleon layers, for example: `GALLEON_PROVISION_LAYERS=cloud-server`
```
Comment (jmesnil) [inline README.md]: Let's use 2 layers to make the example use a comma, something like `GALLEON_PROVISION_LAYERS=cloud-server,postgresql-datasource`

File: README.md [hunk 6]
```diff
@@ -1,2 +1,73 @@
 # jboss-eap-8-openshift-image
  OpenShift container images for Red Hat JBoss Enterprise Application Platform 8
+
+This project defines Images allowing you to build and deploy EAP 8 applications on the cloud.
+
+EAP 8 S2I (Source-To-Image) builder images:
```
Comment (jmesnil) [inline README.md:6]: We should write somewhere that the builder and runtime images are not meant to be mixed (or more positively that builder and runtime images must share the same JDK version)

**Review by jmesnil — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (jfdenise) [reply README.md]: For now yes, the doc will evolve with support for channels

**Comment by jfdenise** [comment]
@jmesnil I fixed your comments.

**Comment by jmesnil** [comment]
@jfdenise thanks

**Pull request merged**